docs/config.md

```diff
@@ -2,16 +2,19 @@
 
 These are the config settings available to jj/Jujutsu.
 
-The config settings are located at `~/.jjconfig.toml`.
+The config settings are located at either `~/.config/jj/config.toml`
```

Comment:
I actually use `~/.config/jj/config.toml` and like it. If `jj` didn't support it, I'd probably have a `~/.jjconfig` be a symlink to `~/.config/jj/config.toml` anyway. I like having all my config in one place.

I will delete the mention of it from here, since that is your preference.

Reply:
If you prefer it there, you're probably not alone, so I'm also fine with keeping it then. I also don't mind mentioning that location in here, but note that it's not `~/.config/jj/config.toml` on all platforms. Feel free to not mention the alternative path, as you suggested.